pom.xml

```diff
@@ -184,7 +184,7 @@
                         <plugin>
                             <groupId>com.qulice</groupId>
                             <artifactId>qulice-maven-plugin</artifactId>
-                            <version>0.15.4</version>
+                            <version>0.16.2</version>
```

Comment:
@mkordas task was upgrade to 0.16.2. So in terms of task definition this version is correct and no puzzle required as for me. I was long time in review but I don't think this is my problem